### PR TITLE
Update CRNN Readme

### PR DESCRIPTION
**Pull request by hqkate**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Updated the following parts:
1. Added the section for MS Lite Inference (section 6)
2. Added the data shape of mindir file in Notes
3. Updated MindIR download links
4. Merged the Accuracy table and Training Perf. table

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/rec/crnn/README_CN.md [hunk 1]
```diff
@@ -79,6 +69,7 @@ Table Format:
 - 如需在其他环境配置重现训练结果，请确保全局批量大小与原配置文件保持一致。
 - 模型所能识别的字符都是默认的设置，即所有英文小写字母a至z及数字0至9，详细请看[4. 字符词典](#4-字符词典)
 - 模型都是从头开始训练的，无需任何预训练。关于训练和测试数据集的详细介绍，请参考[数据集下载及使用](#312-数据集下载)章节。
+- MindIR导出时的输入Shape为(1, 3, 32, 100)
```
Comment (HaoyangLee) [inline configs/rec/crnn/README_CN.md:72]:
```suggestion
- CRNN_VGG7和CRNN_ResNet34_vd的MindIR导出时的输入Shape均为(1, 3, 32, 100)。
```

File: configs/rec/crnn/README_CN.md [hunk 2]
```diff
@@ -39,34 +39,24 @@ Table Format:
...
-
-#### 训练性能
 
-| 设备 | 模型 | 骨干网络 | 数据集 | 参数量 | 单卡批量 | 图模式8卡训练 (s/epoch) | 图模式8卡训练 (ms/step) | 图模式8卡训练 (FPS) |
-| :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: | :----: |
-| Ascend910| CRNN | VGG7 | MJ+ST | 8.72 M | 16 | 2488.82 | 22.06 | 5802.71 |
-| Ascend910| CRNN | ResNet34_vd | MJ+ST | 24.48 M | 64 | 2157.18 | 76.48 | 6694.84 |
 
-#### 推理性能
+### 推理性能
```
Comment (HaoyangLee) [inline configs/rec/crnn/README_CN.md:59]:
```suggestion
### 推理端
```

Comment (HaoyangLee) [reply configs/rec/crnn/README_CN.md:59]: 推理端表格居中

File: configs/rec/crnn/README_CN.md [hunk 3]
```diff
@@ -39,34 +39,24 @@ Table Format:
```
Comment (HaoyangLee) [inline configs/rec/crnn/README_CN.md:39]: 本章标题改为：精度结果 --> 训练端

File: configs/rec/crnn/README.md [hunk 4]
```diff
@@ -39,33 +39,22 @@ According to our experiments, the evaluation results on public benchmark dataset
...
+| **Model** | **Context**  | **Backbone** | **Avg Accuracy** | **Train T.** | **FPS** | **Recipe** | **Download** |
+| :-----: | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: |
+| CRNN      | D910x8-MS1.8-G | VGG7 | 82.03%  |  2445 s/epoch | 5802.71 | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_vgg7.yaml)     | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c-573dbd61.mindir)   |
+| CRNN      | D910x8-MS1.8-G | ResNet34_vd | 84.45%  | 2118 s/epoch | 6694.84 | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_resnet34.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_resnet34-83f37f07.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_resnet34-83f37f07-eb10a0c9.mindir) |
```
Comment (HaoyangLee) [inline configs/rec/crnn/README.md:45]:
```suggestion
| CRNN      | D910x8-MS1.8-G | VGG7 | 82.03%  |  2488.82 s/epoch | 5802.71 | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_vgg7.yaml)     | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c-573dbd61.mindir)   |
| CRNN      | D910x8-MS1.8-G | ResNet34_vd | 84.45%  | 2157.18 s/epoch | 6694.84 | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_resnet34.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_resnet34-83f37f07.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_resnet34-83f37f07-eb10a0c9.mindir) |
```

File: configs/rec/crnn/README_CN.md (hunk 3 above)
Comment (HaoyangLee) [reply configs/rec/crnn/README_CN.md:39]: 训练端章节说明精度评估结果是来自于第3.3节，推理端章节说明结果来自于第6节。

File: configs/rec/crnn/README_CN.md [hunk 5]
```diff
@@ -39,34 +39,24 @@ Table Format:
...
   | CRNN | ResNet34_vd | 94.42% | 94.23% | 93.35% | 92.02% | 75.92% | 70.15% | 87.73% | 86.40% | 76.28% | 73.96% | 84.45% |
   </div>
-</details>
-
-
-### 性能
-
-#### 训练性能
 
-| 设备 | 模型 | 骨干网络 | 数据集 | 参数量 | 单卡批量 | 图模式8卡训练 (s/epoch) | 图模式8卡训练 (ms/step) | 图模式8卡训练 (FPS) |
```
Comment (HaoyangLee) [inline configs/rec/crnn/README_CN.md:64]: 保留该表的结果，合并入上方第一个表

**Comment by hqkate** [comment]
Updated according to the comments, please check, thanks thanks @HaoyangLee

**Review by zhtmike — APPROVED** [review]

File: configs/rec/crnn/README.md [hunk 6]
```diff
@@ -33,49 +33,46 @@ Table Format:
...
   | CRNN | VGG7 | 94.53% | 94.00% | 92.18% | 90.74% | 71.95% | 66.06% | 84.10% | 83.93% | 73.33% | 69.44% | 82.03% |
   | CRNN | ResNet34_vd | 94.42% | 94.23% | 93.35% | 92.02% | 75.92% | 70.15% | 87.73% | 86.40% | 76.28% | 73.96% | 84.45% |
   </div>
-</details>
 
-### Performance
+### Inference Perf.
 
-#### Training Perf.
+The inference end performance test is mainly based on Mindspore Lite, please take a look at [Mindpore Lite Inference](#6-mindspore-lite-inference) for more details.
```
Comment (HaoyangLee) [inline configs/rec/crnn/README.md:59]:
```suggestion
The inference end performance test is mainly based on Mindspore Lite, please take a look at [Mindpore Lite Inference](#6-mindspore-lite-inference) for more details.
```
```suggestion
The inference performance is tested on Mindspore Lite, please take a look at [Mindpore Lite Inference](#6-mindspore-lite-inference) for more details.
```

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**